ui/components/ChipGroup.tsx

```diff
@@ -35,13 +35,20 @@ function ChipGroup({ className, chips = [], onChipRemove, onClearAll }: Props) {
           </Flex>
         );
       })}
-      {chips.length > 0 && <Chip label="Clear All" onDelete={onClearAll} />}
+      {chips.length > 0 && (
+        <Chip
+          label="Clear All"
+          className="filter-options-chip"
```

Comment:
💡 It would be cleaner to have `filter-options-chip` class in `ChipGroup` too if possible. It is not supposed to know about a class name in DataTable.

Reply:
removed class and just moved the bg color into the component itself...not sure why it was added originally. Looked in enterprise too for a reason and I don't see one. Should be okay.